tests/unittests/TFLiteImporterTest.cpp

```diff
@@ -101,6 +119,12 @@ static void loadAndRunModel(std::string modelName, float maxError = 1e-6) {
     // Load reference tensor.
     Tensor refT(outT->getType());
     loadTensor(&refT, refFilename);
+    if (tflitePrintTestTensorsOpt) {
+      llvm::outs() << "Reference Tensor:\n";
+      refT.dump();
```

Comment:
Have you tested that you can receive correct logs using the mix of llvm::outs() and .dump()? Since sometimes .dump() happens after regular printing stream.

Reply:
probably some internal buffering in llvm::outs() or in another library, but here, dump() is ok as it's using llvm::outs().